pytorch_lightning/overrides/torch_distributed.py

```diff
@@ -93,7 +104,7 @@ def _broadcast_noop(obj, *_, **__):
         return obj
 
     broadcast_object_list = _broadcast_noop
-elif _TORCH_GREATER_EQUAL_1_8:
+elif _TORCH_GREATER_EQUAL_1_8 and not _HPU_AVAILABLE:
```

Comment:
Mind confirming the implementation for `broadcast_object_list` is the same on PyTorch master ?